### DISCUSSION TITLE
Setting custom headers in vms-source

### DISCUSSION THREAD

**Discussion by kohazidaniel** [ghettovoice/vuelayers#470] [Q&A] [answered]
Hey guys!

Is it possible to set custom headers (for example authorization) in the requests made by vms-source?

**Comment by kohazidaniel** [comment, marked as answer]
I've succesfully overriden tileLoadFunction, not the cleanest  solution, but works for now

**Reply by ghettovoice** [reply]
Hello @kohazidaniel ,

as I know, your solution with custom tileLoadFunction is actually recommended and easiest way with openlayers. More complex way is implementation of custom source.

**Reply by kohazidaniel** [reply]
As i researched the topic more thoroughly, i totally agree, thank you for getting back to me.
